addons/extension/XEH_serverPostInit.sqf

```diff
@@ -48,12 +48,12 @@ addMissionEventHandler ["MPEnded", {
     private _missionType = getMissionConfigValue QEGVAR(missionTesting,missionType);
     private _playerCount = {isPlayer _x} count allUnits;
 
-    if (_missionType == 2 && _playerCount < 15) then {
+    if (_missionType == 2 && _playerCount > 15) then {
         [
             "potato",
             "embed",
             "arma",
-            "TVT is over.\n Slotting for COOP will be starting up immediately",
+            "TVT is over. Slotting for COOP will be starting up immediately",
```

Comment:
`TvT has concluded. Beginning slotting for COOP.` might be a better phrase but not essential to change